aten/src/ATen/native/mps/operations/BinaryOps.mm

```diff
@@ -22,6 +24,295 @@
 typedef MPSGraphTensor* (^BinaryOpBlock)(BinaryOpCachedGraph*, MPSGraphTensor*, MPSGraphTensor*);
 #define BinaryOpFn(graph, primary, secondary) MPSGraphTensor* (mps::BinaryOpCachedGraph* graph, MPSGraphTensor* primary, MPSGraphTensor* secondary)
 
+enum class BinaryKernelType {
```

Comment:
Can you please move this entire code to a new file like BinaryOpsKernels.mm? 
The BinaryOps.mm is a mess already, and we don't want to make it worse.